betting-game-react/src/components/Login/Login.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Login from './Login';
+
+describe('<Login />', () => {
+  test('it should mount', () => {
```

Comment:
Same comment as above.
I'd rather prefer the "bdd" kind of test namings for example:

`describe('Login component for users', () => {
     test('the login should be visible to the user as soon as the component mounted') ...
`

Reply:
Has been changed :-)